app/src/main/java/org/simple/clinic/sync/ModelSync.kt

```diff
@@ -0,0 +1,9 @@
+package org.simple.clinic.sync
+
+import io.reactivex.Completable
+
+interface ModelSync {
```

Comment:
I agree. I feel that the naming of `DataSync` should be changed, since we already have `PatientSync`, `BloodPressureSync` etc, which do a completely different thing than what `DataSync` itself does. 

What data sync does now is more of a `SyncCoordinator`, which is what I suggest naming it.

Do you have any suggestions?

Reply:
Had a discussion offline and decision was to rename `DataSync` and related classes to `SyncCoordinator`.